Update README.md (#30)
AutoResearch/autora-template-cookiecutter: {{ cookiecutter.__project_slug }}/README.md

```
{% endif -%}

### Step 6: Add the package to [`autora`](https://github.com/autoresearch/autora)

> Once your package is working, and you've published it on PyPI, you can **make a pull request** on 

Reviewer: ```suggestion
Once your package is working and published, you can **make a pull request** on 
```

Reviewer: ^make more open-ended for if/when we extend to Conda

[`autora`](https://github.com/autoresearch/autora) to have it vetted and added to the "parent" package.

> First clone the repository, add the package as optional dependency and add documentation from the package repository.

Reviewer: ```suggestion
To do so, you'll need to clone the parent repository, add your package to it as an optional dependency, and make sure your documentation is imported.
```

> Then create a pull-request with the changes to let us know about your contribution.

Reviewer: ```suggestion
Then create a pull request with the changes to let us know about your contribution.
```

#### Add the package as optional dependency 

> In the `pyorject.toml` [`autora`](https://github.com/autoresearch/autora) package, 

Reviewer: ```suggestion
In the `pyproject.toml` file of your cloned [`autora`](https://github.com/autoresearch/autora) package, 
```

> add an optional dependency for the package in the `[project.optional-dependencies]` section:

Reviewer: ```suggestion
add an optional dependency for your new package in the `[project.optional-dependencies]` section:
```

```toml
example-contribution = ["autora-example-contribution==1.0.0"]
```

!!! success
    Ensure you include the version number.

> Add the example-contribtuion to be part of the corresponding all dependency:

Reviewer: ```suggestion
Add the example-contribution to be part of the corresponding all-contribution-type dependencies:
```

```toml
all-contribution-type = [
    ...
    "autora[example-contribution]",
    ...
]
```

### Add documentation from the package repository

> Import the documentation in the `mkdocs.yml` of the [`autora`](https://github.com/autoresearch/autora) package:

Reviewer: ```suggestion
Import your documentation in the `mkdocs.yml` of the [`autora`](https://github.com/autoresearch/autora) package:
```
